src/ws/ssh.c

```diff
@@ -167,7 +164,7 @@ gssapi_push_creds (CockpitSshData *data)
         goto out;
       }
 
-    g_debug ("setup kerberos credentials in ccache: %s", data->krb5_ccache_name);
+    g_debug ("setup kerberos credentials in ccache: %s", cache_name);
```

Comment:
This doesn't have an existing test. Nothing changed in the functionality, all that is different is cache name now comes from ```data->ssh_options->krb5_ccache_name``` instead of ```data->krb5_ccache_name```. We probably should test this, but I don't think it's worth holding up this PR for.